scripts/sh/ensure-important-modules-up-to-date.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+
+output="$(go list -u -m -json all | jq --raw-output --argjson packages '
+[
+  "github.com/nyaruka/phonenumbers",
+  "golang.org/x/crypto",
+  "golang.org/x/exp",
+  "golang.org/x/image",
+  "golang.org/x/mod",
+  "golang.org/x/net",
+  "golang.org/x/oauth2",
+  "golang.org/x/sync",
+  "golang.org/x/sys",
+  "golang.org/x/telemetry",
+  "golang.org/x/term",
+  "golang.org/x/text",
+  "golang.org/x/time",
+  "golang.org/x/tools",
+  "golang.org/x/vuln"
```

Comment:
If I understand it correctly, `ensure-important-modules-up-to-date` is for forcing upgrade to latest version of packages. And packages which needs to be always latest is only `github.com/nyaruka/phonenumbers` because we need the latest db. What is the reason we want to include other packages here?

Reply:
Updated.